x-pack/plugins/apm/public/components/shared/PropertiesTable/index.js

```diff
@@ -7,56 +7,20 @@
 import React from 'react';
 import styled from 'styled-components';
 import _ from 'lodash';
-import STATIC_PROPS from './staticProperties.json';
-import {
-  units,
-  colors,
-  px,
-  fontFamilyCode,
-  fontSizes,
-  unit,
-  fontSize
-} from '../../../style/variables';
+import { PROPERTY_CONFIG } from './propertyConfig';
+import { colors, units, px, unit, fontSize } from '../../../style/variables';
 import { EuiIcon } from '@elastic/eui';
 
 import { getFeatureDocs } from '../../../utils/documentation';
 import { ExternalLink } from '../../../utils/url';
+import { NestedKeyValueTable } from './NestedKeyValueTable';
+
+const indexedPropertyConfig = _.indexBy(PROPERTY_CONFIG, 'key');
```

Comment:
This is much better than doing `_.find(PROPERTY_CONFIG, { key: currentKey }),` on every invocation! 👍